HopsanCore/include/Nodes.h

```diff
@@ -471,6 +471,27 @@ class NodeModelica : public Node
 };
 
 
+//! @brief A petri net node
+//! @ingroup NodePetriNet
+class NodePetriNet : public Node
+{
+public:
+    //! @brief The data variable indexes, DataLength is used internally
+    //! @ingroup NodePetriNet
+    enum DataIndexEnumT {State, Flow, DataLength};
+    enum DataIndexEnumOldT {STATE, FLOW};
```

Comment:
The DataIndexEnumOldT is not neede for new nodes, better to force users to use the new non-uppercase versions

Reply:
Ok, I was unsure of this, but it sounds reasonable.